pulse/pacat-simple-vchan.c

```diff
@@ -210,6 +215,16 @@ static void process_playback_data(struct userdata *u, pa_stream *s, size_t max_l
         return;
     }
 
+    if (verbose > 1) {
+        pa_usec_t latency;
+        int negative;
+
+        if (pa_stream_get_latency(s, &latency, &negative))
+            pacat_log("pa_stream_get_latency() failed");
+
+        pacat_log("process_playback_data(): vchan data %d max_length %d latency %llu", space_in_vchan, max_length, latency);
```

Comment:
```suggestion
        if (pa_stream_get_latency(s, &latency, &negative))
            pacat_log("pa_stream_get_latency() failed");
        else
            pacat_log("process_playback_data(): vchan data %d max_length %d latency %llu", space_in_vchan, max_length, latency);
```

otherwise we might log uninitialized stack rubble

Reply:
Yes, but a) I still want other values logged, and b) thanks to `PA_STREAM_INTERPOLATE_TIMING` that happens basically only once at the very start.